libkbfs/bare_root_metadata.go

```diff
@@ -871,6 +871,40 @@ func (md *BareRootMetadataV2) FakeInitialRekey(h BareTlfHandle) {
 	md.RKeys = TLFReaderKeyGenerations{rkb}
 }
 
+// GetTLFPublicKey implements the BareRootMetadata interface for BareRootMetadataV2.
+func (md *BareRootMetadataV2) GetTLFPublicKey(keyGen KeyGen) (TLFPublicKey, bool) {
+	if keyGen > md.LatestKeyGeneration() {
+		return TLFPublicKey{}, false
+	}
+	return md.WKeys[keyGen].TLFPublicKey, true
+}
+
+// AreKeyGenerationsEqual implements the BareRootMetadata interface for BareRootMetadataV2.
+func (md *BareRootMetadataV2) AreKeyGenerationsEqual(codec Codec, other BareRootMetadata) (bool, error) {
```

Comment:
When we have V3 we'll have to handle comparing V2s and V3s, right? If so in the future it might be better to have a "KeyGenerations" object or something that we can return and compare. I guess it'll become clearer once we have V3...